Docs/RHCL-RELEASE-GUIDE.md

```diff
@@ -0,0 +1,873 @@
+
+### Step 7: Monitor Release Execution
+
+1. **Check Release Status**
+   ```bash
+   oc get release {release-name} -o yaml
+   ```
+   - Look at `status.conditions`
+   - Note the `processing` and `succeeded` conditions
+
+2. **Find Release PipelineRun**
+   ```bash
+   oc project rhtap-releng-tenant
+   oc get pipelineruns -l release.appstudio.openshift.io/name={release-name}
+   ```
+
+3. **View Pipeline Logs**
+   ```bash
+   oc logs -f {pipelinerun-pod-name}
+   ```
+
+   Or use the Konflux UI to view detailed task logs.
+
+4. **Common Pipeline Tasks**
+   - `verify-enterprise-contract`: Validates against production EC policy
+   - `push-snapshot`: Pushes images to target registry
+   - `sign-images`: Signs images with Red Hat keys
+   - `update-pyxis`: Updates container metadata in Pyxis
+
+### Step 7: Verify Release Success
+
+1. **Check Pipeline Completion**
+   ```bash
+   oc get pipelinerun {pipelinerun-name}
+   ```
+   - Status should be `Succeeded`
+
+2. **Verify Images in Registry**
+   - For staging:
+     ```bash
+     podman pull registry.stage.redhat.io/rhcl-1/rhcl-rhel9-operator:1.2.0
+     ```
+   - For production:
+     ```bash
+     podman pull registry.redhat.io/rhcl-1/rhcl-rhel9-operator:1.2.0
+     ```
+
+3. **Check Pyxis Metadata**
+   - Go to: https://pyxis.engineering.redhat.com/
+   - Search for repository: `rhcl-1/rhcl-rhel9-operator`
+   - Verify new tags are listed
+
+### Step 8: Repeat for Other Components
+
+RHCL releases multiple components together. Repeat the process for:
+- Authorino operator and bundle
+- DNS operator and bundle
+- Limitador operator and bundle
+- Supporting images (wasm-shim, console-plugin)
+
+Each component has its own RPA file in the `api-management` directory.
+
+---
+
+## Step-by-Step: Releasing the File-Based Catalog
+
+The file-based catalog (FBC) publishes operator metadata to OpenShift OperatorHub indexes.
+
+### What is a File-Based Catalog?
+
+A file-based catalog is a YAML file (`catalog.yaml`) that defines:
+- **Packages**: Operator packages (e.g., `rhcl-operator`, `dns-operator`)
+- **Channels**: Update channels (e.g., `stable`, `fast`)
+- **Bundles**: References to operator bundle images
+- **Deprecations**: Deprecated versions or channels
+
+
+### FBC Architecture for RHCL
+
+RHCL maintains separate FBC components for each supported OCP version:
+- **OCP 4.14**: `rhcl-file-based-catalog-4-14`
+- **OCP 4.15**: `rhcl-file-based-catalog-4-15`
+- **OCP 4.16**: `rhcl-file-based-catalog-4-16`
+- **OCP 4.17**: `rhcl-file-based-catalog-4-17`
+- **OCP 4.18**: `rhcl-file-based-catalog-4-18`
+- **OCP 4.19**: `rhcl-file-based-catalog-4-19`
+- **OCP 4.20**: `rhcl-file-based-catalog-4-20`
+
+Each component contains a `catalog.yaml` file in the `rhcl-file-based-catalog/` directory.
+
+### Step 1: Update FBC Catalog YAML
+
+**Important**: FBC catalogs must be updated for each operator across all supported OCP versions. The process is identical for all four operators:
```

Comment:
It's worth mentioning that the different operators support different OCP versions

Authorino Operator is the only one that should be shipped for versions <OCP-4.17